ocs_ci/deployment/aws.py

```diff
@@ -397,6 +417,387 @@ def deploy_ocp(self, log_cli_level='DEBUG'):
         if config.DEPLOYMENT.get('host_network'):
             self.host_network_update()
 
+        if config.ENV_DATA['rhel_workers']:
+            self.add_rhel_workers()
+
+    def gather_worker_data(self):
+        """
+        Gather various info like vpc, iam role, subnet,security group,
+        cluster tag from existing RHCOS workers
+        """
+        suffix = 'no0'
+        stack_name = f'{self.cluster_name}-{suffix}'
+        resource = self.cf.list_stack_resources(StackName=stack_name)
+        worker_id = self.get_worker_resource_id(resource)
```

Comment:
This will always return only 1 resource ID and will lead creation of new workers in single az only
 
![Screenshot (16)](https://user-images.githubusercontent.com/32088367/69724448-ef168b80-1141-11ea-81c1-7ba29bb749b5.png)